exporter/coralogixexporter/config.go

```diff
@@ -36,8 +36,8 @@ type Config struct {
 	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
 	exporterhelper.TimeoutSettings `mapstructure:",squash"`
 
-	// The Coralogix trace ingress endpoint
-	configgrpc.GRPCClientSettings `mapstructure:",squash"`
+	// Coralogix traces ingress endpoint
+	Traces configgrpc.GRPCClientSettings `mapstructure:"traces"`
```

Comment:
Isn't this bringing a breaking change to your users? You should follow the deprecation guidelines instead of just breaking your user's configuration.

Reply:
Breaking changes are allowed, but we have a deprecation policy: https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes

I'm not comfortable approving this PR with known breaking changes ignoring our policy. If other approvers are OK with that, I won't block this.

cc @open-telemetry/collector-contrib-approvers

Reply:
@jpkrohling You are correct, no argue here, but it is based on a request that came from all of our customers using the endpoint, which was communicated and agreed this breaking change is not "breaking change" for their use case, as it is currently only acts as jaeger endpoint. All of them are waiting for the PR to be merged and released. 
I agree that from a technical perspective it should be changed according to the deprecated policy, but from a business perspective it should be enough to add what @povilasv already added in the docs. We can argue the importance of a business perspective when it comes to opensource, but in the end it should have some weight. 

Anyways, we will change it accordingly, but it is important for future reference.

Reply:
Keeping it backward compatible is really, really easy in this case. The effort to discuss this was already higher than implementing the change itself, I'm afraid ;-) Just add the new structure, and check during the bootstrap whether the old one is being used or the new one (or both). If only the old one is used, just copy it over to the new one, logging a warning message asking users to start using the new config property. The remaining of your code can use the new location already. You'd then keep both audiences happy.